pkgs/development/python-modules/cairocffi/default.nix

```diff
@@ -45,11 +45,10 @@ buildPythonPackage rec {
   ];
 
   nativeBuildInputs = [
+    flit-core
     setuptools
```

Comment:
Can you check if `setuptools` can be removed (from here and from the argument list at the top of the file)?